test/rubygems/test_gem_commands_exec_command.rb

```diff
@@ -217,6 +217,7 @@ def test_gem_with_platform_dependencies
 
   def test_gem_with_platform_and_platform_dependencies
     pend "extensions don't quite work on jruby" if Gem.java_platform?
+    pend "terminates on mswin" if Gem.win_platform?
```

Comment:
Is there any reason why this test was passing in local CI but not in Ruby CI?

Reply:
I'm not sure why nobu ignore it. Maybe Ruby core team tested it with native `mswin` platform on AppVeyor, not `mingw` platform.

Reply:
I wonder the same thing, can we also use a ruby_core specific marker, so that we don't lose the coverage ourselves?

Reply:
OK, I investigate it or revert this.

Reply:
Just to clarify, my suggestion was

```ruby
pend "terminates on mswin" if Gem.win_platform? && ruby_repo?
```

just to let the test still run on our CI and make it clear that's ruby-core CI the reason to skip it.

Reply:
Ah, I see. Thanks.